public/app/features/explore/QueryRowActions.tsx

```diff
@@ -35,17 +35,22 @@ export function QueryRowActions(props: Props) {
         </div>
       )}
       <div className="gf-form">
-        <button disabled={!canHide} className="gf-form-label gf-form-label--btn" onClick={onClickToggleHiddenQuery}>
+        <button
+          disabled={!canHide}
+          className="gf-form-label gf-form-label--btn"
+          onClick={onClickToggleHiddenQuery}
+          title="Disable/enable query"
+        >
           <i className={hideQuery ? 'fa fa-eye-slash' : 'fa fa-eye'} />
```

Comment:
Just of curiosity what is the difference between `canHide` and `hideQuery`?

Reply:
Thanks for a great feedback! I have updated the naming of variables and  functions, so it is more more clear and understandable. 
*hideQuery => isDisabled* (if queryResults are disabled/hidden or not).
*canHide => isNotStarted* (this is related to the fact, that if loading state is NotStarted,  then we want to disable the button toggling functionality, as there are no query results. Otherwise it would be toggling between cheatsheet and No Data screen, which wasn't that nice).
*onClickToggleHiddenQuery => onClickToggleDisabled* => as we are replacing _hide_ with _disabled_.

Reply:
Thanks for that clarification @ivanahuckova